src/LibraryManager.Vsix/Json/Completion/VersionCompletionEntry.cs

```diff
@@ -0,0 +1,44 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using Microsoft.VisualStudio.Imaging.Interop;
+using Microsoft.VisualStudio.Language.Intellisense;
+using Microsoft.Web.Editor.Completion;
+using Microsoft.VisualStudio.Text;
+using System.Diagnostics;
```

Comment:
nit: sort

Reply:
removed this line as it's unused now.

---
In reply to: [198905577](https://github.com/aspnet/LibraryManager/pull/258#discussion_r198905577) [](ancestors = 198905577)